bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/handler/EchoHandler.java

```diff
@@ -94,12 +99,12 @@
  * @author Michael Geramb - Initial contribution
  */
 @NonNullByDefault
-public class EchoHandler extends BaseThingHandler {
+public class EchoHandler extends BaseThingHandler implements IAmazonThingHandler {
 
     private final Logger logger = LoggerFactory.getLogger(EchoHandler.class);
     private Gson gson = new Gson();
     private @Nullable Device device;
-    private Set<String> capabilities = new HashSet<>();
+    private Set<String> capabilities = new HashSet<String>();
```

Comment:
Please apply the Diamond operator everywhere

Reply:
Done